### PR TITLE
completed 6-article-comments

### PR DESCRIPTION
**Pull request by mrrobbob**

### CODE REVIEW

File: server/index.controller.js [hunk 1]
```diff
@@ -34,30 +31,40 @@ function getArticleById (req, res, next) {
 function getArticles (req, res, next) {
   selectArticles()
   .then(async (articles) => {
-    const noBody = articles.rows.map((article) => {
-      delete article.body
-      return article
-    })
+    const noBody = articles.rows
     const counts = await db.query(`
```
Comment (nc-hannah-burks) [inline server/index.controller.js:35]: Any querying like this you want instead to be in your model.

File: __tests__/index.test.js [hunk 2]
```diff
@@ -138,4 +139,56 @@ describe('GET /api/articles', () => {
...
+      const comments = res.body.comments
+      expect(Array.isArray(comments)).toBe(true)
+      expect(comments.length).toBe(11)
+      comments.forEach((article) => {
+        expect(article).toMatchObject({
+          comment_id: expect.any(Number),
+          body: expect.any(String),
+          votes: expect.any(Number),
+          author: expect.any(String),
+          article_id: expect.any(Number),
```
Comment (nc-hannah-burks) [inline __tests__/index.test.js:159]: Here we want to check the article_id is 1 specifically, to ensure we are getting the correct article's comments back

File: __tests__/index.test.js [hunk 3]
```diff
@@ -138,4 +139,56 @@ describe('GET /api/articles', () => {
...
+          created_at: expect.any(String)
+        })
+        expect(moment(article.created_at, moment.ISO_8601).isValid()).toBe(true)
```
Comment (nc-hannah-burks) [inline __tests__/index.test.js:162]: Interesting, not see this used before 💡

File: __tests__/index.test.js [hunk 4]
```diff
@@ -138,4 +139,56 @@ describe('GET /api/articles', () => {
...
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then((res) => {
+      const comments = res.body.comments
+      const datesInMs = comments.map((comment) => {
+        return (new Date(comment.created_at)).getTime()
+      })
+      expect(datesInMs).toBeSorted({descending: true})
+    })
+  })
```
Comment (nc-hannah-burks) [inline __tests__/index.test.js:193]: Another test I would like to see added is Status 200, valid ID, but has no comments responds with an empty array of comments. Check your data to see which article id has no comments
